src/plasma/plasma_manager.cc

```diff
@@ -698,17 +710,22 @@ void ignore_data_chunk(event_loop *loop,
   PlasmaRequestBuffer *buf = conn->ignore_buffer;
 
   /* Just read the transferred data into ignore_buf and then drop (free) it. */
-  int done = read_object_chunk(conn, buf);
-  if (!done) {
-    return;
+  int err = read_object_chunk(conn, buf);
```

Comment:
Now that we're handling dropped connections, we don't really need `ignore_data_chunk` anymore, right? E.g., the receiver could simply hang up on the sender if the receiver doesn't want the object that the sender is sending, right?

Not sure which would be more efficient.

Reply:
Hmm, I think we still need it so that we can get any objects that were queued behind the object we want to ignore. I think this case is if we already have the object that's getting transferred, so we just want to ignore the new copy.